transformers/shared/constants/method.go

```diff
@@ -48,6 +48,19 @@ func FlipABI() string {
 		"MCD_FLIP_ZRX_A_1.0.9",
 	})
 }
+func FlipV110ABI() string {
+	return GetABIFromContractsWithMatchingABI([]string{
+		"MCD_FLIP_BAT_A_1.1.0",
+		"MCD_FLIP_ETH_A_1.1.0",
+		"MCD_FLIP_KNC_A_1.1.0",
+		"MCD_FLIP_MANA_A_1.1.0",
+		"MCD_FLIP_TUSD_A_1.1.0",
```

Comment:
Do we need USDT and PAXUSD added here or is that in another PR?

Reply:
I am adding USDT and PAXUSD and #316 😄